lib/ReactViewModels/ViewState.js

```diff
@@ -170,10 +169,11 @@ export default class ViewState {
     }
 
     viewCatalogMember(catalogMember) {
-        if (addedByUser(catalogMember)) {
-            this.userDataPreviewedItem = catalogMember;
-            this.openUserData();
-        } else {
+        if (catalogMember.isUserSupplied === true) {
+          this.userDataPreviewedItem = catalogMember;
+          this.openUserData();
+        }
```

Comment:
Sorry to nitpick formatting, but Terria code should be indented with four spaces and the `else` should be on the same line as the `}`. The spaces, at least, should come automatically from `.editorconfig` if your editor supports it (there's probably a plugin if not).